contrib/babelfishpg_common/src/collation.c

```diff
@@ -33,6 +33,8 @@ static int	server_collation_collidx = NOT_FOUND;
 static Oid	server_collation_oid = InvalidOid;
 static bool db_collation_is_CI_AS = true;
 
+static char *database_collation_name = NULL;
```

Comment:
why can't it be oid?

Reply:
or it could be idx, it will help us reducing word to translate the collation name everytime